File: Seccion 11 - Funciones en C++/Seccion_11_Dart/03_ejercicio_01.dart

import 'dart:io';

double num1, num2;

void pedirDatos() {
  stdout.writeln('Digite un número: ');
  num1 = double.parse(stdin.readLineSync());
  stdout.writeln('Digite otro número: ');
  num2 = double.parse(stdin.readLineSync());
}

void mult(double x, double y) {
  double multiplicacion = x * y;

  print('La multilicacion es: $multiplicacion');
}

void main() {

  pedirDatos();
  mult(num1, num2);
}

File: Seccion 11 - Funciones en C++/Seccion_11_Python/03_ejercicio_01.py

num1 = 0
num2 = 0

def pedir_datos():
    global num1
    global num2
    num1 = float(input('Digite un número: '))
    num2 = float(input('Digite otro número: '))

def mult(x, y):
    multiplicacion = x * y

    print('La multiplicación es: {}'.format(multiplicacion))

pedir_datos()
mult(num1, num2)